website/static/js/logActionsList.json

```diff
@@ -61,6 +61,7 @@
     "retraction_initiated":     "${user} initiated withdrawal of registration of ${node}",
     "retraction_initiated_no_user": "A withdrawal of registration of ${node} is proposed",
     "retraction_approved":  "${user} approved withdrawal of registration of ${node}",
```

Comment:
This is the relevant log here, and per @UdayVarkhedkar 's suggestion, we can go ahead and clean this log up, make the grammar better than we found it.   We can also improve the grammar in the corresponding log in `website/static/js/anonymousLogActionsList.json` which are the logs we show when a user is viewing a registration through an anonymous view-only link.